Common/OpenCL/Filters/itkGPURecursiveGaussianImageFilter.hxx

```diff
@@ -51,6 +51,13 @@ GPURecursiveGaussianImageFilter< TInputImage, TOutputImage >
     = this->m_GPUKernelManager->GetContext()->GetDefaultDevice().GetLocalMemorySize();
 
   this->m_DeviceLocalMemorySize = ( localMemSize / 3 ) / sizeof( float );
+  
+  /**Original implementation set BUFFSIZE as (local memory size)/3, which can handle up to 1365 pixels along one dimension*/
+  /**But in Device with larger local memory size(eg. GTX 2080Ti with 48KB local memory, BUFFZISE=4096), the BUFFSIZE would become*/
+  /**overkill for most medical implementation, and would use up all GPU global memoy during kernel excution;*/
+
+  this->m_DeviceLocalMemorySize = ( this->m_DeviceLocalMemorySize > 1024 )?( 1024 ):( this->m_DeviceLocalMemorySize );
```

Comment:
Please consider using `std::min(a, b)` instead of `a > b ? b : a`.

Reply:
This change makes much sense. @N-Dekker could you implement the std::min in a cleaned up PR?